recipes/keras-nlp/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "keras-nlp" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/keras-nlp-{{ version }}.tar.gz
+  sha256: 44723deed5c271078cae48ca91cf8e8f6c9ffe85e01a52aee17d02366de959a5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - absl-py
+    - numpy
+    - packaging
+    - tensorflow  >=2.0
```

Comment:
Upstream seems to require more: https://github.com/keras-team/keras-nlp/blob/3e9edab1032d1525541e4f57c8e6e31309265465/setup.py#L45

Reply:
@BastianZim, thx for the feedback. I am aware of this dependency, but it looks like that recent PRs went through.

- https://github.com/conda-forge/staged-recipes/pull/17613
- https://github.com/conda-forge/tensorflow-feedstock/pull/294

Will it be possible to install it via `pip`? Otherwise, a feedstock of `tensorflow-text` would make sense for NLP.

Reply:
Unfortunately, we require all dependencies to be available on conda-forge and listed here so we would have to wait for that.

Reply:
I understand that ... in case of avoiding `tensorflow-text` via [`bazel`](https://github.com/tensorflow/text/blob/master/oss_scripts/install_bazel.sh) would that ok like for [catboost-feedstock]( https://github.com/conda-forge/catboost-feedstock/blob/036182496875e787b7d3cbb1c7e051c1c05b395b/recipe/bld.bat)?



```shell
IF "%PY_VER%"=="2.7" (
	%PYTHON% -m pip install --no-deps https://pypi.org/packages/cp27/c/catboost/catboost-%PKG_VERSION%-cp27-none-win_amd64.whl
)

IF "%PY_VER%"=="3.5" (
	%PYTHON% -m pip install --no-deps https://pypi.org/packages/cp35/c/catboost/catboost-%PKG_VERSION%-cp35-none-win_amd64.whl
)

IF "%PY_VER%"=="3.6" (
	%PYTHON% -m pip install --no-deps https://pypi.org/packages/cp36/c/catboost/catboost-%PKG_VERSION%-cp36-none-win_amd64.whl
```

Reply:
No, it needs to be properly installed as a dependency. 
You can also add it to conda-forge yourself, if you need the package?

Reply:
So first setup open a new feedstock for `tensorflow-text` and with package from scratch not like `catboost`?

For the new version of `AutoKeras` they add `keras-nlp` as dependency, while it depends on `tenso flow-text`

https://github.com/conda-forge/autokeras-feedstock/pull/1#issue-1561033951

Reply:
See also: https://github.com/conda-forge/staged-recipes/issues/19578#issue-1297879614

Reply:
Yes, basically submit another PR for tensorflow-text or have it as an output in tensorflow. Not sure what the build process is there. 

I'm not familiar with catboost but we always build from source. These exceptions are only made in very rare cases, where it is impossible to build it differently.